### PR TITLE
Bump FormatUtility.csproj to .NET 5.0

### PR DESCRIPTION
**Pull request by mattboroi**
Update FormatUtility.csproj from netcoreapp3.1 to net5.0, latest version of .NET.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/samples?pullRequest=4319) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/samples?pullRequest=4319) before we can accept your contribution.<br/><br/>:x: mattboroi [sign now](https://cla.dotnetfoundation.org/dotnet/samples?pullRequest=4319)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/samples?pullRequest=4319) it.</sub>

File: windowsforms/formatting-utility/cs/readme.md [hunk 1]
```diff
@@ -16,11 +16,11 @@ The .NET Formating Utility (Formatter.exe) is a .NET Core Windows Forms applicat
 
 ## Sample prerequisites
 
-This sample is written in C# and targets .NET Core 3.0 running on Windows. It requires the [.NET Core 3.0 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.0).
+This sample is written in C# and targets .NET Core 5.0 running on Windows. It requires the [.NET Core 5.0 SDK](https://dotnet.microsoft.com/download/dotnet-core/5.0).
```
Comment (RussKie) [inline windowsforms/formatting-utility/cs/readme.md:19]:
```suggestion
This sample is written in C# and targets .NET 5.0 running on Windows. It requires the [.NET 5.0 SDK](https://dotnet.microsoft.com/download/dotnet-core/5.0).
```

File: windowsforms/formatting-utility/cs/readme.md [hunk 2]
```diff
@@ -16,11 +16,11 @@ The .NET Formating Utility (Formatter.exe) is a .NET Core Windows Forms applicat
...
 
 ## Building the sample
 
-The source code includes an MSBuild project file for C# (a .csproj file) that targets .NET Core 3.0. After you download the .zip file containing the example code, create a directory and Download the sample code files to your computer. To build the example:
+The source code includes an MSBuild project file for C# (a .csproj file) that targets .NET Core 5.0. After you download the .zip file containing the example code, create a directory and Download the sample code files to your computer. To build the example:
```
Comment (RussKie) [inline windowsforms/formatting-utility/cs/readme.md:23]:
```suggestion
The source code includes an MSBuild project file for C# (a .csproj file) that targets .NET 5.0. After you download the .zip file containing the example code, create a directory and Download the sample code files to your computer. To build the example:
```

@adegeo this paragraph reads verbose and repetitive.

**Review by IEvangelist — APPROVED** [review]
Looks good to me, we'll :shipit: - thank you 🙏

**Pull request merged**